README.md

```diff
@@ -66,6 +66,24 @@ var myObject MyInfo
 f.Fuzz(&myObject) // Type will correspond to whether A or B info is set.
 ```
 
+You can use this library for easier [go-fuzz](https://github.com/dvyukov/go-fuzz)ing.
+go-fuzz provides the user a byte-slice, which should be converted to different inputs
+for the tested function. This library can help convert the byte slice. Consider for
+example a fuzz test for a the function `mypackage.MyFunc` that takes an int arguments:
+```go
+// +build gofuzz
+package mypacakge
+
+import "github.com/google/go-fuzz"
+
+func Fuzz(data []byte) int {
+        var i int
+        fuzz.NewFromGoFuzz(data).Fuzz(&i)
+        MyFunc(i)
+        return 0
+}
+```
+
 See more examples in ```example_test.go```.
```

Comment:
I might put the new section below this line, since the examples are not of the new feature.